ui-src/src/components/GameOver.css

```diff
@@ -1,27 +1,242 @@
-  text-shadow: -5px 0 0 #fdff2a,
-    -10px 0 0 #df4a42,
-    5px 0 0 #91fcfe,
-    10px 0 0 #4405fc;
-  animation: blink 1s steps(4, start) infinite;
+  text-shadow: 1px 1px 2px #e3e4f3,
+    0 0 25px #21acba,  /* 5d0b0d */
+    0 0 5px #91fcfe,
+    0 0 10px #4405fc;
+  animation: blink 1s steps(4, start) 3;
 }
 
+/* 1px 1px 2px black, 0 0 25px blue, 0 0 5px darkblue; */
+
  @keyframes blink{
    to{
      visibility: hidden
    }
 }
+
+.game-over-title {
+  flex: 1 1;
+}
+
+
+/* ----------------------  GameOver Stats Screen ---------------- */
+
+.stats-jumbotron {
+  padding-left: 60px;
+  padding-right: 60px;
+  border-radius: 6px;
+  margin-bottom: 30px;
+  display: block;
+}
+
+.stats-header {
+  font-size: 63px;
+  font-weight: 500px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  line-height: 1.1;
+  display: block;
+  left: 0;
+  right: 0;
+  top: 0.8em;
+  bottom: 0;
+  margin: auto;
+  position: absolute;
+  background: -webkit-linear-gradient(#6600ff, #150a4e);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+  animation: title 2s 4s 1 ease both;
+}
+
+@keyframes title {
+  0% {
+    transform: translateZ(500px);
+    opacity: 0;
+  }
+  50% {
+    opacity: 0;
+  }
+  100% {
+    transform: translateZ(0px);
+    opacity: 1;
+  }
+}
+
+.stats-overlay {
+  overflow: hidden;
+  position: absolute;
+  top: 20px;
+  left: 2.5%;
+  width: 95%;
+  height: 795px;
+  margin: 0 auto;
+  background: rgba(0, 0, 0, .92);
+  border: 5px dotted slateblue;
+  z-index: 99999;
+  animation: end-game-stats 3s 3.8s 1 ease both;
+}
+
+@keyframes end-game-stats {
+  0% {
+    top: -100px;
+    opacity: 0;
+  }
+  100% {
+    top: 20px;
+  }
+}
+
+.stats-info {
+  font-weight: 400;
+  font-size: 30px;
+  color: slateblue;
+  border: transparent;
+  background: transparent;
+  display: inline-block;
+  margin: 100px auto 15px;
+  cursor: pointer;
+}
+.stats-header {
+  margin-top: 30px;
+  margin-bottom: 50px;
+  width: 100%;
+  font-size: 3rem;
+  line-height: 40px;
+  letter-spacing: 3px;
+  background: -webkit-linear-gradient(#eee, #4148b1);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+.circle {
+  z-index: 201;
+  color: black;
+  background: black;
+	border-radius: 50%;
+  border: 1.5px solid #eec02c;
+  height: 7em;
+  width: 7em;
+  left: calc(50% - 3em);
+  bottom: 17.6em;
+  position: absolute;
+}
+
+.holochain-logo {
+  z-index: 200;
+  height: 13em;
+  width: 13em;
+  left: 50%;
+  margin-left: -6em;
+  bottom: 14.6em;
+  position: absolute;
+  animation: spin 7s 2 linear;
+  animation-delay: 0.1s;
+}
+
+@keyframes spin {
+    from {
+        transform:rotate(0deg);
+    }
+    to {
+        transform:rotate(360deg);
+    }
+}
+
+.winner-jdenticon {
+  width: 75px;
+  height: 75px;
+  display:block;
+  margin: 0 auto;
+}
+
+.winner {
+  z-index: 300;
+  text-align: center;
+  padding: 12px;
+  min-width: 150px;
+}
+
+.winner p {
+  justify-content: center;
+  margin: 5px;
+}
```

Comment:
Let's experiment with putting the name over the jdenticon and including 1st, 2nd, 3rd as text